src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -322,29 +322,29 @@ export function getChartData(
   contactTracingData: any;
 } {
   const rtRangeData =
-    projection &&
-    projection.rt &&
-    projection.getDataset('rtRange').map(d => ({
-      x: d.x,
-      y: d.y?.rt,
-      low: d.y?.low,
-      hi: d.y?.high,
-    }));
+    projection?.rt == null
```

Comment:
same with the == catching undefine dhere

Reply:
So here it is definitely intentional / necessary because `projection?.rt` will return `undefined` if `projection` is `null` (It's just how the optional chaining operator works. I don't love it).